composer.json

```diff
@@ -30,7 +30,14 @@
     "require": {
         "php": ">=8.0",
         "ext-swoole": "*",
-        "utopia-php/servers": "0.1.*"
+        "utopia-php/servers": "0.1.*",
+        "utopia-php/compression": "0.1.*"
+    },
+    "repositories": {
+        "utopia-php/compression": {
+            "type": "git",
+            "url": "https://github.com/utopia-php/compression"
+        }
```

Comment:
This can be removed now that the package was published.

Reply:
done